var/spack/repos/builtin/packages/dla-future-fortran/package.py

```diff
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+# dlaf-no-license-check
```

Comment:
I guess this doesn't belong here?

Reply:
I thought about removing it, but since it's an harmless comment and I would have to remove it at every update, I leaned towards leaving it in. But I can remove it if that's preferred.

Reply:
I'm indifferent, I was just surprised to see it there. Up to you.